pkg/etcdenvvar/etcd_env.go

```diff
@@ -78,6 +79,12 @@ func getFixedEtcdEnvVars(envVarContext envVarContext) (map[string]string, error)
 		"ETCD_DATA_DIR":              "/var/lib/etcd",
 		"ETCD_QUOTA_BACKEND_BYTES":   "7516192768", // 7 gig
 		"ETCD_INITIAL_CLUSTER_STATE": "existing",
+		"ASSET_DIR" : "/home/core/assets",
```

Comment:
keep these local to the scripts.  They don't vary.  Some are even specific to a single script.